input/posts/en/transfer_learning.md

```diff
@@ -0,0 +1,37 @@
+﻿title: Transfer learning
+lead: Convolutional neural network training (CNN)
```

Comment:
This should be brief text about "what is going on". Explain the whole post in (approx) two sentences. If you are able to explain it in this way that means you understand it well..